independent-projects/arc/runtime/src/main/java/io/quarkus/arc/Unremovable.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.arc;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates that the class marked with this annotation should never be removed by Arc as a bean
```

Comment:
"Indicates that a bean marked with this annotation should never be removed."?

Reply:
Fixed